here-naksha-storage-http/src/test/java/com/here/naksha/storage/http/cache/RequestSenderCacheTest.java

```diff
@@ -95,7 +95,6 @@ void testOneId() {
             TimeUnit.HOURS
     );
 
-    assertEquals(PROP_ID_1, PROP_ID_1_COPY);
```

Comment:
I don't understand why this and the 127 line was deleted

Reply:
As I'm switching to a static class from a record, these are no longer the same object. Discussion for switching from record to class in another chat thread here.

Reply:
As that other problem is resolved, reverting this change